packages/server/package.json

```diff
@@ -150,13 +150,6 @@
   "funding": [
     "https://trpc.io/sponsor"
   ],
-  "optionalDependencies": {
-    "@types/aws-lambda": "*",
-    "express": "*",
-    "fastify": "*",
-    "next": "*",
-    "ws": "*"
-  },
```

Comment:
confirm that rollup doesn't bundle these deps now. ik some bundlers will bundle dev deps if they're not in peerDependencies (not sure if optionalDependencies work the same)

Reply:
looks like something like that (not quite but still broken) is happening 
![CleanShot 2024-11-22 at 13 06 00@2x](https://github.com/user-attachments/assets/bc993187-229c-46e6-b2ee-4ff16718d7ad)